addons/crm_livechat/models/chatbot_script_step.py

```diff
@@ -62,4 +70,39 @@ def _process_step_create_lead(self, discuss_channel):
         create_values.update(self._chatbot_crm_prepare_lead_values(
             discuss_channel, customer_values['description']))
 
-        self.env['crm.lead'].create(create_values)
+        return self.env['crm.lead'].create(create_values)
```

Comment:
Isn't part of the spec about specifically creating a lead assigning it to an available person in priority?
I'm not seeing anything related to that in the current PR.